packages/atlas/src/components/_navigation/SidenavViewer/FollowedChannels.styles.ts

```diff
@@ -17,31 +17,51 @@ export const ChannelsTitle = styled(Text)`
   margin-bottom: ${sizes(4)};
   padding-left: ${NAVBAR_LEFT_PADDING}px;
   width: ${EXPANDED_SIDENAVBAR_WIDTH - NAVBAR_LEFT_PADDING}px;
-  color: ${oldColors.gray[300]};
+  color: ${cVar('colorText')};
+  text-transform: uppercase;
+  letter-spacing: 0.07em;
 `
+
 export const ChannelsWrapper = styled.div`
-  padding-left: ${NAVBAR_LEFT_PADDING}px;
   width: ${EXPANDED_SIDENAVBAR_WIDTH}px;
   overflow-y: auto;
   overflow-x: hidden;
   margin-bottom: 10px;
 `
 
 export const ChannelsList = styled.ul`
-  width: ${EXPANDED_SIDENAVBAR_WIDTH - NAVBAR_LEFT_PADDING}px;
+  color: ${cVar('colorText')};
+  width: ${EXPANDED_SIDENAVBAR_WIDTH}px;
   overflow-x: hidden;
   padding: 0;
   margin: 0;
 `
+
 export const StyledChannelLink = styled(ChannelLink)`
+  width: 100%;
+  height: ${sizes(6)};
```

Comment:
Why do we set a fixed height here? This should be determined by inner padding/spacing